src/ui/js/Game.js

```diff
@@ -2699,11 +2699,11 @@ Game.gamePlayerStatus = function(player, reversed, game_active) {
  * The return object contains a boolean saying whether the die is clickable,
  * and a string describing the reason why or why not.
  *
- * @param  object  die            The die to be displayed
- * @param  boolean die_status     Status of the die ('active' or 'captured')
- * @param  string  player         Whose is the die? ('player' or 'opponent')
- * @param  boolean player_active  Is the player displaying the die active?
- * @return object
+ * @param  {object}  die            The die to be displayed
+ * @param  {boolean} die_status     Status of the die ('active' or 'captured')
+ * @param  {string}  player         Whose is the die? ('player' or 'opponent')
+ * @param  {boolean} player_active  Is the player displaying the die active?
+ * @returns {object}
```

Comment:
This is nit-level, but there were previously two spaces after each param so that the param types would line up with the return types.  Can you add a third space after each param so they line up with the 'returns' types?